cou/commands.py

```diff
@@ -153,6 +153,22 @@ def get_subcommand_common_opts_parser() -> argparse.ArgumentParser:
         type=batch_size_arg,
         default=1000,
     )
+    subcommand_common_opts_parser.add_argument(
+        "--purge",
+        help="Delete data from shadow tables. before cloud upgrade.\n"
+        "Default to disable purge.",
+        action=argparse.BooleanOptionalAction,
+        default=False,
+    )
+    subcommand_common_opts_parser.add_argument(
+        "--purge-before",
+        dest="purge_before",
+        help="Specifying –before will delete data from all shadow tables "
+        "that is older than the date provided. "
+        "Date strings may be fuzzy, such as 'Oct 21 2015'. "
```

Comment:
Please add the formats with examples of what is accepted. By the [docs](https://docs.openstack.org/nova/2024.1/cli/nova-manage.html#db-purge):

> Accepts date strings in the default format output by the date command, as well as YYYY-MM-DD[HH:mm:ss].

Reply:
Updated